eth/api.go

```diff
@@ -1111,7 +1112,7 @@ func (tx *Tx) UnmarshalJSON(b []byte) (err error) {
 		tx.GasLimit = rpc.NewHexNumber(0)
 	}
 	if tx.GasPrice == nil {
-		tx.GasPrice = rpc.NewHexNumber(defaultGasPrice)
+		tx.GasPrice = rpc.NewHexNumber(50000000000)
```

Comment:
I don't know. This is a hotfix and not a refactor. Perhaps a next PR can address this. I'm not entirely sure what this struct is for.